docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/setinputfiles.md

```diff
@@ -70,3 +71,80 @@ export default async function () {
 ```
 
 {{< /code >}}
+
+### Local File Example
+
+{{< code >}}
+
+```javascript
+import { browser } from 'k6/experimental/browser';
+import encoding from 'k6/encoding';
+import { open } from 'k6/experimental/fs';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+// Declare the location of the file on the local filesystem.
+let file;
+(async function () {
+  file = await open('/abs/path/to/file.txt');
+})();
+
+export default async function () {
```

Comment:
Should we use `try` `catch`?

Reply:
Yes, to be consistent, it makes. And, yep, I know what you mean and agree it's worth discussing. I created an issue about this behavior once: https://github.com/grafana/xk6-browser/issues/814.

Reply:
Changed in 74ac66c183c4e84edd044d6fdbc036f68b974bdc